packages/documentation-spa/docs/microservices/feedback/feedback-service.md

```diff
@@ -3,138 +3,233 @@ id: feedback-service
-![image alt text](image_8.png)
-
-**getFeedbackBy:** get feedback by any of its properties like id, title, description for more detail refer docs in graphql playground.
-![image alt text](image_9.png)
-
-## Target Users
-
-This docs is meant for developers as well end users.
-
-## Applications (SPAs)
-
-It is used by the feedback-spa for listing feedback and adding feedback
+#### Prerequisites
+1. **NodeJS** should be installed (_version\&gt;=__v14.15.4_)
+2. **NPM** should be installed _(version\&gt;= __6.4.1__ )_
+3. Version control system required. Preferably **git**.
+
+#### Steps
+
+1. Clone the [repository](https://github.com/1-Platform/one-platform).
+
+```sh
+git clone git@github.com:1-Platform/one-platform.git
+```
+
+2. Switch the working directory to the user microservice
+
+```sh
+cd one-platform/packages/feedback-service
+```
+
+3. Install the microservice dependencies.
+
+```sh
+npm i
+```
+### Start
+
+1. Run npm start:dev to run your microservice for dev env and npm start for production env.
+2. Navigate to port 8080 to see the microservice.
+
+eg: http://localhost:8080/graphql
+
+### Build
+
+1. [Webpack](https://webpack.js.org/) is used for the build system in the microservices.
+2. Run npm build:dev to generate a build for dev env and npm build for production build.
+
+### Testing
+
+1. For testing microservice with [supertest](https://www.npmjs.com/package/supertest) with the preconfigured settings.
+2. Test related environment configurations are present in .test.env under the e2e folder.
+3. Execute the command for testing.
+
+```sh
+npm run test
+```
+
+## API References
+
+In the GraphQL GET Operations are defined as Queries and POST/PUT/PATCH operations are defined as Mutations.
+
+### Queries
+#### List all Feedbacks
+
+- Operation Name - ListFeedbacks
+
+Example Query
+```bash
+query ListFeedbacks {
+  listFeedbacks {
+    _id
+    summary
+    description
+    experience
+    error
+    config
+    state
+    ticketUrl
+    category
+    source
+    module
+    assignee{
+      name
+      uid
+      email
+    }
+    createdOn
+    createdBy {
+      name
+      uid
+      email
+    }
+    updatedOn
+    updatedBy {
+      name
+      uid
+    }
+  }
+}
+```
+@@### List Feedback
+
+- Operation Name - ListFeedback
+- Variables - ```_id```
+
+Example Query
+```bash
+query ListFeedbacks($_id: ID!) {
+  listFeedback(_id: $_id) {
+    _id
+    summary
+    description
+    experience
+    error
+    config
+    state
+    ticketUrl
+    category
+    source
+    module
+    assignee{
+      name
+      uid
+      email
+    }
+    createdOn
+    createdBy {
+      name
+      uid
+      email
+    }
+    updatedOn
+    updatedBy {
+      name
+      uid
+    }
+  }
+}
+```
+
+### Mutations
+#### Create feedback
+1. Create new feedback to database and open the ticket accordingly.
+
+- Operation Name - CreateFeedback
+
+- Variables - _FeedbackInput_
+
+Example Mutation
+```bash
```

Comment:
```suggestion
```graphql
```

Reply:
AFAIK there is no official support for markdown in graphql